app/javascript/components/App.jsx

```diff
@@ -1,15 +1,36 @@
 import React, { Component } from 'react';
 import Home from './Home';
+import SignUp from './SignUp';
+import SignIn from './SignIn';
+import Axios from 'axios';
 import {HashRouter as Router, Route, NavLink, Switch} from 'react-router-dom'
 
 class App extends Component {
+  handleSignOut = (e) =>{
+    e.preventDefault();
+    Axios({
+      method: 'delete',
+      url: '/auth/sign_out',
+      data: JSON.parse(sessionStorage.user)
+    }).then(function (response) {
+        window.location = "/"
+      }).catch(function(error) {
+        window.location = "/#/sign_out"
```

Comment:
I'm not sure that is a good idea. you are trying to perform 'GET' request which was originally 'DELETE'. there is no such route which matches delete /sign_out

Reply:
it's typo, in this line need write url "/"